app/controllers/crops_controller.rb

```diff
@@ -215,6 +215,15 @@ def destroy
   private
 
   def crop_params
-    params.require(:crop).permit(:en_wikipedia_url, :name, :parent_id, :creator_id, :approval_status, :request_notes, :reason_for_rejection, :rejection_notes, scientific_names_attributes: [:scientific_name, :_destroy, :id])
+    params.require(:crop).permit(:en_wikipedia_url,
```

Comment:
I really hate this indentation style, and would much prefer

```
params.require(:crop).permit(
  :en_wikipedia_url,
  :name,
  # etc
)
```
But I'm prepared to be out-voted. What do other people think?